app/views/catalog/_previous_next_doc.html.erb

```diff
@@ -0,0 +1,8 @@
+<div class="row search-links">
+  <div class="col-md-10">
+    <%= render(Blacklight::SearchContextComponent.new(search_context: @search_context, search_session: search_session)) %>
+  </div>
+  <div class="col-md-2">
+    <%= link_back_to_catalog label: 'Back to search', class: 'btn btn-default' %>
```

Comment:
I would not put this in this partial, because the momentum has been to do away with overridable partials and switch to components.

Reply:
But to invoke the new component don't you still need to override this partial?

Reply:
Not if blacklight does away with this partial. I'd prefer we move this into the show page.